ledger-service/http-json/src/it/scala/http/HttpServiceWithPostgresIntTest.scala

```diff
@@ -20,11 +20,14 @@ class HttpServiceWithPostgresIntTest
   override def wsConfig: Option[WebsocketConfig] = None
 
   "query persists all active contracts" in withHttpService { (uri, encoder, _, _) =>
+    val (party, headers) = partyAndAuthHeaders("Ian")
+    val searchDataSet = genSearchDataSet(party)
```

Comment:
Where is `searchDataSet` used?

Reply:
apologies this is a confusing change but its used on line `26` below